boards/beagle/beagleplay_cc1352/beagleplay-cc1352-pinctrl.dtsi

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Vaishnav Achath
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/cc13xx_cc26xx-pinctrl.h>
+
+&pinctrl {
+	/* UART0 */
+	uart0_tx_default: uart0_tx_default {
+		pinmux = <13 IOC_PORT_MCU_UART0_TX>;
+		bias-disable;
+	};
+	uart0_rx_default: uart0_rx_default {
+		pinmux = <12 IOC_PORT_MCU_UART0_RX>;
+		bias-disable;
+		input-enable;
+	};
+
+	/* On-board antenna pinmux states */
```

Comment:
The antenna isn't "On-board". This pinmux feeds J9, where an external SubG antenna provided with BeaglePlay should be connected.